src/network/serverpackethandler.cpp

```diff
@@ -1346,20 +1348,37 @@ void Server::handleCommand_RemovedSounds(NetworkPacket* pkt)
 	}
 }
 
+static void sanitize_field(std::string &s)
+{
+	// truncate on NULL
+	auto pos = s.find('\000');
```

Comment:
```suggestion
	auto pos = s.find('\0');
```

(is there a particular reason you prefer `\000` here? would make sense in a string literal imo, but not as useful in a character literal)

Reply:
no particular reason